test/time_feature/test_lag.py

```diff
@@ -206,7 +210,8 @@
     + [335, 336, 337, 503, 504, 505, 671, 672, 673, 719, 720, 721],
     # centered around each of the last 7 days (delta = 1) + last 4 weeks (delta = 1) + last 1 month (delta = 1) +
     #  last 8th and 12th weeks (delta = 0)
-    "6H": [
+    "6"
+    + H: [
```

Comment:
Wondering if `("6" + H): [` would be kept in one line by black (similar for the changes that follow)